src/main/java/com/io/image/manager/controller/ImageController.java

```diff
@@ -130,6 +130,7 @@ public ImageController(ImageService imageService, CacheRecordRepository cacheRep
 
         return ResponseEntity.ok()
                 .header("ttl", String.valueOf(cacheResult.getTTL()))
+                .header("Content-Type","image/"+extension)
```

Comment:
Can you please format the code properly? IntelliJ has built in formatter that you can run on the whole project with one click. It just hurts to watch when you don't apply spaces after coma and with operators.